js/two_block.js

```diff
@@ -0,0 +1,118 @@
+//  1)
+const helloOleg = function(value) {
+
+    if(value === 'Mark') {
+        console.log(`Hi, ${value}`)
+    } else {
+        console.log(`Hello, ${value}`)
+    }
+}
```

Comment:
your function must return value, not just log

Reply:
Corrected